### DISCUSSION TITLE
Persist color mode when changing MacOS theme

### DISCUSSION THREAD

**Discussion by neildotvn** [chakra-ui/chakra-ui#2311] [Q&A]
How can I persist color mode when changing MacOS theme from light to dark? Or disable the color mode?

I notice the color mode value in localStorage of the documentation doesn't change when MacOS theme is changed. But my app doesn't have the same effect.

OS: MacOS Cataline 10.15.7
Chakra-ui: 1.0.0-rc.3
Nextjs: 9.5.2

**Comment by ljosberinn** [comment]
Hi, would you mind upgrading to `rc.7`? This should be fixed now, there were various bugs before that.
